selfservice/flow/settings/hook.go

```diff
@@ -282,7 +282,8 @@ func (e *HookExecutor) PostSettingsHook(w http.ResponseWriter, r *http.Request,
 		WithField("flow_method", settingsType).
 		Debug("Completed all PostSettingsPrePersistHooks and PostSettingsPostPersistHooks.")
 
-	trace.SpanFromContext(r.Context()).AddEvent(events.NewSettingsSucceeded(r.Context(), i.ID, string(ctxUpdate.Flow.Type), ctxUpdate.Flow.Active.String()))
+	trace.SpanFromContext(r.Context()).AddEvent(events.NewSettingsSucceeded(
+		r.Context(), i.ID, string(ctxUpdate.Flow.Type), ctxUpdate.Session.AMR))
```

Comment:
Isn't `ctxUpdate.Session.AMR` the method used to authenticate the session?

I would expect that field to contain `"profile"`, `"password"`, etc. depending on which group the user updated in the settings.

Reply:
Yes, it is the method used in the session. Should it be the group that got updated? Maybe that makes more sense, but it is not clear.

Reply:
Yes, I think it should be.